herddb-core/src/main/java/herddb/sql/CalcitePlanner.java

```diff
@@ -27,6 +27,9 @@
 import java.util.logging.Logger;
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
+
+import herddb.core.*;
```

Comment:
Please avoid star imports

Reply:
Done. I am sorry about that. I think my IntelliJ does it without asking me about it sometimes.